docs/index.rst

```diff
@@ -241,6 +272,186 @@ Dialogue Summaries
+  along with the generated dialogue ``id`` field and ``url`` field for
+  accessing the project description.
+
+  .. TODO mention that the user creating the project is given admin access once
+  we have documented permission levels
+
+  .. sourcecode:: http
+
+     POST /projects/ HTTP/1.1
+     Content-Type: application/json
+
+     {
+       "title": "Maternal Health ZA",
+     }
+
+  .. sourcecode:: http
+
+     HTTP/1.1 201 Created
+     Content-Type: application/json
+
+     {
+       "id": "23",
+       "url": "/projects/23",
+       "title": "Maternal Health ZA",
+       "dialogues": [],
+       "is_archived": false
+     }
+
+.. _projects-put:
+
+.. http:put:: /projects/(str:project_id)
+
+  Replaces the :ref:`description <data-projects>` of the project with id ``project_id`` with the description given in the request body and returns the given description, along with the projects's ``id`` and the ``url`` for accessing the projects's description.
+
+  .. sourcecode:: http
+
+     PUT /projects/23 HTTP/1.1
+     Content-Type: application/json
+
+     {
+       "id": "23",
+       "title": "Maternal Health ZA"
+     }
+
+  .. sourcecode:: http
+
+     HTTP/1.1 200 OK
+     Content-Type: application/json
+
+     {
+       "id": "23",
+       "title": "Maternal Health ZA"
+       "url": "/projects/23",
+       "is_archived": false,
+       "dialogues": [{
+         "id": "21",
+         "title": "Service Rating Survey",
+         "url": "/projects/23/dialogues/21",
+         "is_archived": false,
+         "is_published": false,
+         "has_changes": false
+       }]
+     }
+
+.. _projects-patch:
+
+.. http:patch:: /projects/(str:project_id)/
+
+  Partially updates the :ref:`description <data-projects>` of the project with id ``project_id`` with the :ref:`instructions <overview-partial-updates>` in the request body and returns the given description, along with the projects's ``id`` and the ``url`` for accessing the project's description.
+
+.. sourcecode:: http
+
+   PATCH /projects/23 HTTP/1.1
+   Content-Type: application/json-patch+json
+
+   [{
+     "op": "replace",
+     "path": "/title",
+     "value": "Maternal Health ZA"
+   }]
+
+.. sourcecode:: http
+
+   HTTP/1.1 200 OK
+   Content-Type: application/json
+
+   {
+     "id": "23",
+     "title": "Maternal Health ZA"
+     "url": "/projects/23",
+     "is_archived": false,
+     "dialogues": [{
+       "id": "21",
+       "title": "Service Rating Survey",
+       "url": "/projects/23/dialogues/21",
+       "is_archived": false,
+       "is_published": false,
+       "has_changes": false
+     }]
+   }
+
+
+Archiving projects
+~~~~~~~~~~~~~~~~~~
+A project can be archived by setting ``is_archived`` to ``true`` when :ref:`replacing <projects-put>` or :ref:`partially updating <projects-patch>` the project description. The project is still accessible via the api, but its dialogues will no longer be triggered by any events that were previously associated to them.
```

Comment:
A bit unsure about this, still wondering if there is something better we can do for archived projects and dialogues. This is the best I can think of for now, but will give this some more thought.